TinyConstraints/Classes/Constrainable.swift

```diff
@@ -36,12 +36,12 @@ extension TinyView: Constrainable {
     }
 }
 
-extension LayoutGuide: Constrainable {
+@MainActor extension LayoutGuide: Constrainable {
     @discardableResult
     public func prepareForLayout() -> Self { return self }
 }
 
-public protocol Constrainable {
+@MainActor public protocol Constrainable {
```

Comment:
Strongly recommend the following instead:

```suggestion
@preconcurrency @MainActor 
public protocol Constrainable {
```

This makes it so that existing libraries that don't yet have strict concurrency enabled can use this library version without having to make changes to their code.